ush/templates/FV3.input.yml

```diff
@@ -241,7 +241,7 @@ FV3_GFS_v15p2:
     fhcyc: 0.0
     fhlwr: 3600.0
     fhswr: 3600.0
-    fhzero: 6.0
+    fhzero: 1.0
```

Comment:
Should we just remove this as is done in the other sections?

Reply:
Yes, I meant to do that after testing. Good catch.